R/download_data.R

```diff
@@ -16,20 +17,20 @@
 #' # Download the origin-destination on district level for the a date range in March 2020
 #' spod_download_data(
 #'   type = "od", zones = "districts",
-#'   date_range = c("2020-03-20", "2020-03-24")
+#'   dates = c(start = "2020-03-20", end = "2020-03-24")
 #' )
 #'
 #' # Download the origin-destination on district level for select dates in 2020 and 2021
 #' spod_download_data(
 #'   type = "od", zones = "dist",
-#'   dates_list = c("2020-03-20", "2020-03-24", "2021-03-20", "2021-03-24")
+#'   dates = c("2020-03-20", "2020-03-24", "2021-03-20", "2021-03-24")
 #' )
 #'
 #' # Download the origin-destination on municipality level using regex for a date range in March 2020
 #' # (the regex will capture the dates 2020-03-20 to 2020-03-24)
 #' spod_download_data(
 #'   type = "od", zones = "municip",
-#'   date_regex = "2020032[0-4]"
+#'   dates = "2020032[0-4]"
```

Comment:
Great that now works.

Reply:
Do we need to update the `@param dates` description?

Reply:
I believe I did https://github.com/Robinlovelace/spanishoddata/blob/15fbb32e6c4773080e196d02dc4c7bad659b8939/R/internal_utils.R#L5 it's there because it is then inherited by many functions in the pacakge